src/landpatterns/VirtualLP.stanza

```diff
@@ -393,6 +363,33 @@ public defn add-artwork (
     VirtualArtwork(ls, sh, class = class)
   append-all(vp, new-art)
 
+doc: \<DOC>
+  Add a cutout to the landpattern with a given shape.
+<DOC>
+public defn add-cutout (
+  vp:VirtualLP,
+  shape:Shape
+  --
+  name:String = ?,
+  class:Seqable<String>|String = []
+  ) -> False:
+  append(vp, VirtualArtwork(Cutout(), shape, name? = name, class = class))
+
+doc: \<DOC>
+  Add keepout regions to the landpattern with a sequence of given shapes.
+<DOC>
+public defn add-keepouts (
```

Comment:
Why make `add-cutout` singular and `add-keepouts` plural with a sequence ? 

These inconsistencies are not good as part of a public facing API. 

Check the `add-copper` implementation - it has polymorphics that support both. This is the way these interfaces should be added.